cle/loader.py

```diff
@@ -119,8 +121,7 @@ def close(self):
     def __repr__(self):
         if self._main_binary_stream is None:
             return '<Loaded %s, maps [%#x:%#x]>' % (os.path.basename(self._main_binary_path), self.min_addr(), self.max_addr())
-        else:
-            return '<Loaded from stream, maps [%#x:%#x]>' % (self.min_addr(), self.max_addr())
+        return '<Loaded from stream, maps [%#x:%#x]>' % (self.min_addr(), self.max_addr())
```

Comment:
I disabled this particular pylint check (`no-else-return`) as soon as I upgraded my pylint, so if pylint told you to do this, grab the new pylintrc from angr-dev!